src/components/_internal/BaseToastBanner/BaseToastBanner.tsx

```diff
@@ -0,0 +1,87 @@
+import React from 'react';
+import styled from 'styled-components';
+import { isNotNilOrEmpty } from 'ramda-adjunct';
+
+import { BaseToastBannerVariants } from './BaseToastBanner.enums';
+import { IconTypes, SSCIconNames } from '../../../theme/icons/icons.enums';
+import { getColor, pxToRem } from '../../../utils';
+import { ColorTypes } from '../../../theme/colors.enums';
+import { Padbox } from '../../layout/Padbox';
+import { Inline } from '../../layout/Inline';
+import { Icon } from '../../Icon';
+import {
+  BaseToastBannerProps,
+  BaseToastBannerWrapperProps,
+} from './BaseToastBanner.types';
+import { SpaceSize } from '../../../theme/space.types';
+
+export const baseToastBannerColorVariants = {
+  [BaseToastBannerVariants.info]: ColorTypes.info500,
+  [BaseToastBannerVariants.warn]: ColorTypes.warning500,
+  [BaseToastBannerVariants.error]: ColorTypes.error500,
+  [BaseToastBannerVariants.success]: ColorTypes.success500,
+};
+
+const iconVariants = {
+  [BaseToastBannerVariants.info]: SSCIconNames.infoCircle,
+  [BaseToastBannerVariants.warn]: SSCIconNames.exclTriangleSolid,
+  [BaseToastBannerVariants.error]: SSCIconNames.errorSolid,
+  [BaseToastBannerVariants.success]: SSCIconNames.checkCircleSolid,
+};
+
+const IconPadbox = styled(Padbox)<{
+  $variant?: BaseToastBannerProps['variant'];
+}>`
+  background-color: ${({ $variant }) =>
+    getColor(baseToastBannerColorVariants[$variant])};
+  display: flex;
+  align-items: center;
+`;
+
+const IconWrapper = styled.div<{ $size: SpaceSize }>`
+  display: flex;
+  align-items: center;
+  width: ${({ $size }) => pxToRem($size)};
+  height: ${({ $size }) => pxToRem($size)};
```

Comment:
You are passing sting value into the `pxToRem` method, that does not convert string values into numbers